mysql-test/include/filter_file.inc

```diff
@@ -114,7 +123,7 @@ perl;
   {
     chomp;
     s/\015?\012?$//;
-    if (!%column_names)
+    if (!%column_names && $skip_column_names == 0)
```

Comment:
Guys, with such rules our work will stop. There is no need to me mtr guru to understand this code, it's just perl. This changes allows to optionally skip parsing the first line of a file. And filter_file.inc contains quite detailed comments, the examples of it's usage can be found with grep.